src/server/pfs/fuse/options.go

```diff
@@ -1,32 +1,45 @@
 package fuse
 
-import "github.com/hanwen/go-fuse/fuse/nodefs"
+import (
+	"github.com/hanwen/go-fuse/v2/fs"
+
+	"github.com/pachyderm/pachyderm/src/client/pkg/errors"
+	"github.com/pachyderm/pachyderm/src/server/pkg/uuid"
+)
 
 // Options is for configuring fuse mounts. Any of the fields may be left nil
 // and `nil` itself is a valid set of Options which uses the default for
 // everything.
 type Options struct {
-	Fuse *nodefs.Options
-	// commits is a map from repos to commits, if a repo is unspecified then
-	// the master commit of the repo at the time the repo is first requested
-	// will be used.
-	Commits map[string]string
+	Fuse *fs.Options
+
+	// Write indicates that the pfs mount should allow writes.
+	// Writes will be written back to the filesystem.
+	Write bool
```

Comment:
I couldn't find where setting `Write = false` would prevent writes to PFS but I'm probably missing something.

Reply:
Nope you're right, good catch, It was used in a previous iteration that required a lot more setup to support writes. I'll hook this up so that without it you get a read only filesystem.